cpp/Sorting Algorithms/Bubble Sort.cpp

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+using namespace std;
+
+void swap(int &a, int &b)
+{
+	int temp = a;
+	a = b;
+	b = temp;
+}
+
+void bubbleSort(int *arr, int n)
+{
+	for (int i = 0; i < n; i++)
+		for (int j = i + 1; j < n; j++)
+			swap(arr[i], arr[j]);
```

Comment:
Hi, @OjusWiZard Thanks for your contribution. Almost there but need a small change!
In bubble sort, we only swap the elements if the next element is of lower value than the current one, else we start for the next index.
Adding and if statement to check this will work out.

Reply:
Ohh you're right!
Can't believe that I forgot that... 😶 
On it!